.doc_gen/metadata/glacier_metadata.yaml

```diff
@@ -75,8 +94,43 @@ glacier_UploadArchive:
               snippet_tags:
                 - python.example_code.glacier.GlacierWrapper
                 - python.example_code.glacier.UploadArchive
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/glacier
+          sdkguide: sdk-for-javascript/v3/developer-guide/glacier-example-uploadarchive.html
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - glacier.JavaScript.createclientv3
+            - description: Upload the archive.
+              snippet_tags:
+                - glacier.JavaScript.upload.uploadArchiveV3
+        - sdk_version: 2
+          github: javascript/example_code/glacier
+          sdkguide: sdk-for-javascript/v2/developer-guide/glacier-example-uploadrchive.html
+          excerpts:
+            - description:
+              snippet_tags:
+                - glacier.JavaScript.upload.uploadArchive
   services:
     glacier: {UploadArchive}
+glacier_UploadMultipartPart:
+  title: Create a multipart upload out of 1 megabyte chunks of a Buffer object using an &AWS; SDK
+  title_abbrev: Create a multipart upload
+  synopsis: create a multipart upload to an &GLlong; vault.

```

Comment:
I'd shorten the title and move the example-specific info the example description.
```
title: Create a multipart upload to an &GL; vault using an &AWS; SDK.
```

Reply:
implemented